WordCounter/Program.cs

```diff
@@ -3,18 +3,41 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.IO;
+using System.Diagnostics;
 
 namespace WordCounter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            string fileName = args[0];
-            string text = System.IO.File.ReadAllText(fileName);
+            // Run Tests
```

Comment:
You don't want to have Test stuff in your main project.  In the .net world you'll add a new Test project to your solution.  You can choose from NUnit tests, XUnit tests, or MSTest.  For now I would just create a new MSTest project.  Call it WordCounter.Tests